Glossary.event handlers (#258)
cylc/cylc-doc: src/glossary.rst

         It is also possible to have multiple :term:`flows <flow>` running in the
         scheduler :term:`schduler` simultaneously.

   event handlers
   handlers

>       Carry out an action when the Cylc scheduler detects that an arbitrary

Reviewer: ```suggestion
      A command or action to execute when the Cylc scheduler detects that a particular 
```

      event has occurred. This allows Cylc to centralize automated handling of

>       critical events. Cylc can do may things when it detects an event.

Reviewer 2: s/may/many

      Use-cases include:

>       - Send an email message
>       - Run a Cylc command

Reviewer: ```suggestion
      - Send an email message
      - Run a Cylc command
      - Run any user-specified script or command
```
